ScoutingDataOutline.json

```diff
@@ -1,20 +1,23 @@
 {
-    "event_code": "",
+    "event_code": "eventCode",
```

Comment:
Why are you hard coding the value of event code to the string "eventCode". If this is just to prevent something else from breaking from an empty string, you should probably handle the error better. Alternatively, consider values such as "unknown". That way the default is meaningful to a user.

Reply:
This sets the event code based on the event page that they loaded